docs/usage/administration/monthly_active_users.md

```diff
@@ -0,0 +1,73 @@
+# Monthly Active Users
+
+Synapse can be configured to record the number of monthly active users (also referred to as MAU) on a given homeserver.
+For clarity's sake, MAU only tracks local users.
+
+Please note that the metrics recorded by the [Homeserver Usage Stats](/usage/administration/monitoriing/reporting_homeserver_usage_statistics)
+are calculated differently. The `monthly_active_users` does not take into account any of the rules below, and counts any users 
+who have made a request to the homeserver in the last 30 days.
+
+See the [configuration manual](/usage/configuration/config_documentation.html#limit_usage_by_mau) for details on how to configure MAU.
```

Comment:
We should probably link from the configuration manual back to this document too?